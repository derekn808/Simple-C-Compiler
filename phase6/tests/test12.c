int main(void)
{
    char *p, s[100];



    scanf("%s", s);
    p = s;

    while(*p) {
	if (*p >= *"a" && *p <= *"z")
	    *p = *p + *"A" - *"a";
	p = p + 1;
    }

    printf("%s\n", s);
}
