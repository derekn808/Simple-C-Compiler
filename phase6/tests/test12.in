Segmentation_fAULT_(core_dump)
