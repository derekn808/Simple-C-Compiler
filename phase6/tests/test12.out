SEGMENTATION_FAULT_(CORE_DUMP)
